src/frequenz/client/base/channel.py

```diff
@@ -120,6 +138,40 @@ def parse_grpc_uri(
         parsed_uri.netloc if parsed_uri.port else f"{parsed_uri.netloc}:{defaults.port}"
     )
 
+    keep_alive = (
+        defaults.keep_alive.enabled
+        if options.keep_alive is None
+        else options.keep_alive
+    )
+    channel_options = (
+        [
+            ("grpc.http2.max_pings_without_data", 0),
+            ("grpc.keepalive_permit_without_calls", 1),
+            (
+                "grpc.keepalive_time_ms",
+                (
+                    (
+                        options.keep_alive_interval
+                        if options.keep_alive_interval is not None
+                        else defaults.keep_alive.interval
```

Comment:
For consistency with `keep_alive` definition
```suggestion
                        defaults.keep_alive.interval
                        if options.keep_alive_interval is None
                        else options.keep_alive_interval
```